RadixWallet/Clients/LocalAuthenticationClient/LocalAuthenticationClient+Live.swift

```diff
@@ -26,44 +26,33 @@ extension LAContext {
 	private func canEvaluate(
 		policy: LAPolicy,
 		errorHandling: (LAError) -> Result<Bool, Error>?
-	) async throws -> Bool {
-		try await withCheckedThrowingContinuation { [weak self] (continuation: CheckedContinuation<Bool, Swift.Error>) in
-			guard let self else {
-				continuation.resume(throwing: Error.contextDeinitialized)
-				return
-			}
-			var error: NSError?
-			let canEvaluate = self.canEvaluatePolicy(policy, error: &error)
+	) throws -> Bool {
+		var error: NSError?
+		let canEvaluate = self.canEvaluatePolicy(policy, error: &error)
 
-			guard let evaluationError = error else {
-				continuation.resume(returning: canEvaluate)
-				return
-			}
+		guard let evaluationError = error else {
+			return canEvaluate
+		}
 
-			guard let laError = evaluationError as? LAError else {
-				let reason = String(describing: error)
-				continuation.resume(throwing: Error.evaluationFailedWithOtherError(reason: reason))
-				return
-			}
+		guard let laError = evaluationError as? LAError else {
+			let reason = String(describing: error)
+			throw Error.evaluationFailedWithOtherError(reason: reason)
+		}
 
-			guard let result = errorHandling(laError) else {
-				continuation.resume(returning: canEvaluate)
-				return
-			}
+		guard let result = errorHandling(laError) else {
+			return canEvaluate
+		}
 
-			switch result {
-			case let .success(canEvaluate_):
-				continuation.resume(returning: canEvaluate_)
-				return
-			case let .failure(error):
-				continuation.resume(throwing: error)
-				return
-			}
+		switch result {
+		case let .success(canEvaluate_):
+			return canEvaluate_
+		case let .failure(error):
+			throw error
 		}
```

Comment:
Isn't this the same as `return try result.get()`?

Reply:
yes, I think we had some logging there earlier... fixing